ProjectLighthouse.Servers.Website/Pages/Layouts/BaseLayout.cshtml.cs

```diff
@@ -26,15 +25,16 @@ public class BaseLayout : PageModel
     public string Title = string.Empty;
 
     private UserEntity? user;
+
     public BaseLayout(DatabaseContext database)
     {
         this.Database = database;
 
         this.NavigationItems.Add(new PageNavigationItem(BaseLayoutStrings.HeaderUsers, "/users/0", "user friends"));
         this.NavigationItems.Add(new PageNavigationItem(BaseLayoutStrings.HeaderPhotos, "/photos/0", "camera"));
         this.NavigationItems.Add(new PageNavigationItem(BaseLayoutStrings.HeaderSlots, "/slots/0", "globe americas"));
-        
-        this.NavigationItemsRight.Add(new PageNavigationItem(GeneralStrings.Announcements, "/announce", "bullhorn"));
+
+        this.NavigationItemsRight.Add(new PageNavigationItem(GeneralStrings.Notifications, "/notifications", "bell"));
```

Comment:
Idk how much work this would entail but a notification button should have some kind of indicator as to whether or not you have notifications like a little red dot above it or something.